dynamic/testdata/TestPrimitiveTypes/diff(some).golden

```diff
@@ -2,6 +2,7 @@
   "changes": "DIFF_SOME",
   "diffs": [
     "attrNumberComputed",
+    "attrNumberRequired",
```

Comment:
I think this is a correct result, this used to be a bug. Likely a degenerate case because a required property cannot be not specified in the program.